### ISSUE TITLE
Change Length and Width of land tool independently

### ISSUE THREAD

**Issue by chrismilne300** [OpenRCT2/OpenRCT2#3463]
If you wanted a long narrow piece of land changing at the moment you would have to change it one square at a time. It would be good to be able to change the length and width of the brush independently.

I've done  a mock up of how I see it working.
![untitled-7781](https://cloud.githubusercontent.com/assets/1381723/14937502/097bcf54-0f01-11e6-8680-e139564cac29.jpg)

**Comment by PFCKrutonium** [comment]
I think a spinner might work better, but otherwise I like this idea.

**Comment by marijnvdwerf** [comment]
Or a selection area like Word uses for tables.
![image](https://cloud.githubusercontent.com/assets/737603/14938084/79ba9ac6-0f19-11e6-9652-6521286f0846.png)

Or making it a two step interaction: first select the land area, then move it.

**Comment by spacek531** [comment]
I'm not a huge fan because I like it being square.

**Comment by jensj12** [comment]
Perhaps just change the input box to take two values instead of one so the current widget (what's that kind of selection thing called?) can stay for those who like it, and change it to something more informational when the brush is not square.

**Comment by chrismilne300** [comment]
What about adding a button that allows the user to select the shape of the brush, then displaying the setting necessary for the shape of the bush.

As explained better by me mock up below.

![shape](https://cloud.githubusercontent.com/assets/1381723/14952726/c6f3a728-105a-11e6-9442-bf808dd9a85d.jpg)

This way we keep the current widget but add the option should the user want.

**Comment by janisozaur** [comment]
Gimp's linked and unliked states:
![zaznaczenie_114](https://cloud.githubusercontent.com/assets/550290/14952903/0916b1ea-1065-11e6-88a8-6aa2bc3aeb38.png)


![zaznaczenie_115](https://cloud.githubusercontent.com/assets/550290/14952908/1464f764-1065-11e6-8400-ee47a197baa8.png)

**Comment by duncanspumpkin** [comment]
With the surface paint merge we can actually change the logic for it to do rectangles if so required.

**Comment by IntelOrca** [comment]
I don't think you would really need a rectangle selection if you instead had a "flatten" tool, which you could drag around, smoothed or non-smoothed.
